### DISCUSSION TITLE
Does Preview Mode require fallback:true?

### DISCUSSION THREAD

**Discussion by tomgreeEn** [vercel/next.js#13535] [Help]
Hi there,

I've been implementing Preview Mode for my site which mainly uses GetStaticProps, and after a few days of going around in circles, I've discovered for posts that had not been published at the time of the latest build, for preview mode to work you need to have fallback mode enabled. 

This is not mentioned anywhere that I could see in the docs so I'm posting here to:

a) check that this is expected behaviour (all the demos seem to have fallback: true)
b) help anyone else having the same issue.

Thanks

Tom

**Comment by timneutkens** [comment]
> for preview mode to work you need to have fallback mode enabled.

Preview mode works with and without fallback: true. However it only covers *previews* and not *publishing* the content. How are you deploying your app?

**Comment by njw275** [comment]
I've been working with this for a couple days now and have not gotten it to work with `fallback: false` but could with `fallback: true`. If someone has an example of it working with false that would be great. Just commenting to help anyone else with this issue when they see it
